std/os/index.zig

```diff
@@ -633,16 +633,21 @@ fn posixExecveErrnoToErr(err: usize) PosixExecveError {
     };
 }
 
-pub var linux_aux_raw = []usize{0} ** 38;
+pub var linux_elf_aux_maybe: ?[*]std.elf.Auxv = undefined;
```

Comment:
it looks like you want this to be initialized to `null`, not `undefined`.

Reply:
but it is always set at program initialization, initializing it to null requires more code in every executable. I make it a maybe because one might want to unmap the entire elf auxiliary vector to save space.

Reply:
It is not set at program initialization when zig code is used as a library. Why would initializing it to null require more code? It's static data, so if it's null it would go in the .bss section.